app/src/main/java/com/example/parkingproject/utils/DateTimeUtils.java

```diff
@@ -0,0 +1,9 @@
+package com.example.parkingproject.utils;
+
+import java.util.Date;
+
+public class DateTimeUtils {
```

Comment:
As this is a utility class with all static methods, make it `final` so it can't be extended and add a private empty constructor so that it can't be instantiated.